terraform/aws/security/aws-network-acl-allows-all-ports.yaml

```diff
@@ -33,8 +33,8 @@ rules:
           metavariable: $NETWORK_ACL
           regex: ^(aws_network_acl|aws_default_network_acl)$
     - metavariable-regex:
```

Comment:
Ah one more thing, you need to un-indent the last `metavariable-regex` as a whole by 2 spaces